tests/functional/registry/conftest.py

```diff
@@ -3,17 +3,17 @@
 
 @pytest.fixture
 def create_token(gov, Token):
-    def create_token():
-        return Token.deploy({"from": gov})
+    def create_token(decimals):
```

Comment:
Do this instead, and you don't need to change so much
```suggestion
    def create_token(decimals=18):
```